lib/httparty/connection_adapter.rb

```diff
@@ -116,6 +117,12 @@ def connection
       if options[:timeout] && (options[:timeout].is_a?(Integer) || options[:timeout].is_a?(Float))
         http.open_timeout = options[:timeout]
         http.read_timeout = options[:timeout]
+
+        if RUBY_VERSION >= "2.6.0"
+          http.write_timeout = options[:timeout]
+        else
+          Kernel.warn("Warning: option :write_timeout requires Ruby version 2.6.0 or later")
```

Comment:
It's a timeout for everything, we should set `write_timeout` if we can, but there is no need to add warning here.